buildpacks/ruby/CHANGELOG.md

```diff
@@ -7,7 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- The buildpack now implements Buildpack API 0.10 instead of 0.9, and so requires `lifecycle` 0.17.x or newer. ([#283](https://github.com/heroku/buildpacks-ruby/pull/283/files#commit-suggestions))
+### Changed
```

Comment:
Good spot. I missed this in the last PR review, doh.

Reply:
I wrote down an idea about adding a formatting check step and Colin seems supportive https://github.com/heroku/buildpacks/discussions/23.